src/js/enactors.js

```diff
@@ -0,0 +1,54 @@
+/*
+Copyright 2015 OCAD University
+
+Licensed under the Educational Community License (ECL), Version 2.0 or the New
+BSD license. You may not use this file except in compliance with one these
+Licenses.
+
+You may obtain a copy of the ECL 2.0 License and BSD License at
+https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
+*/
+
+(function ($, fluid) {
+
+    "use strict";
+
+    fluid.registerNamespace("gpii.firstDiscovery.enactor");
+
+    /*
+     * Langauge enactor
+     */
+    fluid.defaults("gpii.firstDiscovery.enactor.lang", {
+        gradeNames: ["fluid.prefs.enactor", "autoInit"],
+        preferenceMap: {
+            "locale": {
```

Comment:
Shouldn't this also use a namespaced name too like `gpii.firstDiscovery.locale`. We should probably use a common terms name for the language i guess, but that's probably okay for now.